### ISSUE TITLE
SAL-POS-POS-ADD-[Tally sheet format and print out after receipts]

### ISSUE THREAD

**Issue by A2000-ISTEAM** [a2000-erp-team/WEBERP#852]
![image.png](https://images.zenhubusercontent.com/5e8e8265b7c1d9906c2bbc2c/b30b1ee1-2bcd-4840-a615-33a082a626e9)

25. Tally Sheet format and print out after receipts.

a) Tally Sheet format
b) A setting to activate this tally sheet print out as it is not very common for market practice (For SSVP internal control requirement only)
c) Tally sheet to be printed after receipts print out.

**Comment by RekhaBhatt** [comment]
Hi @adriantoh ,
![image.png](https://images.zenhubusercontent.com/5e4f83ebbf2772ab3ef41702/1cc6ceaf-365e-4586-9007-41cda0e120d8)
May I know if the item code from your picture above is Description column from this picture?
Thanks

**Comment by RekhaBhatt** [comment]
@adriantoh ,
Another concern is  u said this report is print after receipt is print right? so this report paper size is the same as receipt size? 
Thanks.

**Comment by A2000-ISTEAM** [comment]
Yes the item code is the description in transaction column (product master both code and description same).

Yes receipt size because it is not practical to get another printer to print this tally sheet separately.

**Comment by RekhaBhatt** [comment]
Ok. understood.. thanks

**Comment by A2000-ISTEAM** [comment]
The Tally Sheet title is not printed.

**Comment by A2000-ISTEAM** [comment]
Tally Sheet title still not printed!

**Comment by A2000-ISTEAM** [comment]
Ok done.
